### PR TITLE
[Facebook Adapter] Move app ids to constant value

### PR DESCRIPTION
**Pull request by matiasroldan6**
## Description
Moved the values for the app Ids to a constant on top for easiness of deployment.

### CODE REVIEW

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.PrimaryTestBot/Bots/PrimaryBot.cs [hunk 1]
```diff
@@ -15,6 +15,11 @@ namespace Microsoft.Bot.Builder.Adapters.Facebook.TestBot.Bots
 {
     public class PrimaryBot : ActivityHandler
     {
+        /// <summary>
+        /// Id value for the intended secondary receiver app.
+        /// </summary>
+        private readonly string secondaryReceiverAppId = "<SECONDARY RECEIVER APP ID>";
```
Comment (ceciliaavila) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.PrimaryTestBot/Bots/PrimaryBot.cs]: Shouldn't this be a constant?

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.SecondaryTestBot/Bots/SecondaryBot.cs [hunk 2]
```diff
@@ -15,6 +15,11 @@ namespace Microsoft.Bot.Builder.Adapters.Facebook.SecondaryTestBot.Bots
 {
     public class SecondaryBot : ActivityHandler
     {
+        /// <summary>
+        /// Id value for the intended primary receiver app.
+        /// </summary>
+        private readonly string primaryReceiverAppId = "<PRIMARY RECEIVER APP ID>";
```
Comment (ceciliaavila) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.SecondaryTestBot/Bots/SecondaryBot.cs]: Same here

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.PrimaryTestBot/Bots/PrimaryBot.cs (hunk 1 above)
Comment (matiasroldan6) [reply tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.PrimaryTestBot/Bots/PrimaryBot.cs]: yes

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.SecondaryTestBot/Bots/SecondaryBot.cs (hunk 2 above)
Comment (matiasroldan6) [reply tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.SecondaryTestBot/Bots/SecondaryBot.cs]: yes

**Pull request merged**